game-core/src/test/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcherTest.java

```diff
@@ -46,15 +46,6 @@ void shouldReturnPrimaryValueWhenPrimaryValueSet() {
       assertThat(getLobbySystemProperty(KEY), is(VALUE));
     }
 
-    @Test
-    void shouldClearPrimaryValueWhenPrimaryValueSet() {
-      givenPrimaryValueSet();
-
-      getLobbySystemProperty(KEY);
-
-      assertThat(System.getProperty(KEY), is(nullValue()));
-    }
```

Comment:
What about reversing this test now that the clear statement is removed?

Reply:
It's okay to delete tests sometimes :smile: 
On another level, it's pretty special that the lobby watcher cleared a system property, I don't think we need to be explicit about it being left alone. Ideally we would move this check to code that is testing where we do now have the property clear, though that is a bit prohibitive/not worth it.